### PR TITLE
 Consider maintenance in PostgreSQL and Redis alerts.

### PR DESCRIPTION
**Pull request by zugao**
* Consider maintenance in PostgreSQL and Redis alerts.
* Update appcat dependencies to get instance count from the SLI Prober.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.27.0
+        tag: add-ha-prober
```
Comment (Kidswiss) [inline class/defaults.yml:42]: Please don't forget to change this.

File: component/slos.libsonnet [hunk 2]
```diff
@@ -82,14 +82,36 @@ local prometheusRule(name) =
         },
       },
     ],
+    'vshn-postgresql-ha': [
+      newSLO('uptime', 'vshn-postgresql-ha', params.slos.vshn.postgres.uptime) {
+        description: 'Uptime SLO for High Available PostgreSQL by VSHN',
+        sli: {
+          events: {
+            // The  0*rate(...) makes sure that the query reports an error rate for all instances, even if that instance has never produced a single error
+            error_query: '(sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNPostgreSQL", instances="3"}[{{.window}}]) or 0*rate(appcat_probes_seconds_count{service="VSHNPostgreSQL"}[{{.window}}])) by (service, namespace, name, organization, sla) or vector(0)) > 0',
```
Comment (Kidswiss) [inline component/slos.libsonnet:91]: And what's with clusters that have `instances=2`? They are still HA.

**Review by Kidswiss — CHANGES_REQUESTED** [review]
I think I just found a flaw with your implementation how to determine HA instances. Please also see the comment in: https://github.com/vshn/appcat/pull/81

File: component/slos.libsonnet (hunk 2 above)
Comment (zugao) [reply component/slos.libsonnet:91]: Ah missed that, will update right  away

Comment (zugao) [reply component/slos.libsonnet]: Updated to` !="1"`

**Comment by zugao** [comment]
Changed from _instances (int)_ to _ha (bool)_

**Review by Kidswiss — APPROVED** [review]
LGTM

**Pull request merged**